microprofile/opentracing-smallrye/pom.xml

```diff
@@ -90,6 +90,11 @@
             <scope>provided</scope>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.resteasy</groupId>
+            <artifactId>resteasy-client</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
This is more of a guess, but should we be using:
```
<dependency>
    <groupId>org.eclipse.microprofile.rest.client</groupId>
    <artifactId>microprofile-rest-client-api</artifactId>
    <scope>provided</scope>
</dependency>
```

Reply:
I actually thought about it, but SmallRye's TCK isn't using it. As this is a "last minute" change, I was afraid to use it and have unknown consequences.

Could we open a JIRA to address this for the next version?

Reply:
Yes this can absolutely wait. Honestly we may end up having to spend some time on figuring out module matrix's anyway.